packages/core/src/views/account-center/Index.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+  import { onDestroy } from 'svelte'
+  import { updateAccountCenter } from '../../store/actions'
+  import type { AccountCenter } from '../../types'
+  import Maximized from './Maximized.svelte'
+  import Minimized from './Minimized.svelte'
+
+  export let settings: AccountCenter
+
+  const accountCenterPositions = {
```

Comment:
Would a centered option be useful?

Reply:
Depends what you mean by customizable positioning? If you mean exact spacing from the top or right etc, I don't think it is worth having specific values there. We also need to know the position (ie topRight) so that we can do animations correctly as well for future UI elements that will be added either on top or beneath depending on the position.

Reply:
Totally makes sense